src/tap/utils.py

```diff
@@ -219,56 +226,154 @@ def source_line_to_tokens(obj: object) -> Dict[int, List[Dict[str, Union[str, in
     return line_to_tokens
 
 
```

Comment:
The three functions above are public but are no longer used in the code at all. Should they just stay there?